delivery_total_weight_from_packaging/models/stock_quant_package.py

```diff
@@ -1,32 +1,43 @@
 # Copyright 2021 Camptocamp SA
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class StockQuantPackage(models.Model):
     _inherit = "stock.quant.package"
 
+    shipping_weight = fields.Float(compute="_compute_shipping_weight", readonly=False)
```

Comment:
Add a comment saying it's an override? Could you explain again what is the purpose of `readonly=False` here? To not break tests which set a value on it?

Reply:
the purpose is pretty simple: I  don't want to change its default behavior. People expect to be able to customize it.
The wizard itself is going to update that value. It just makes sense IMO :)

Reply:
@simahawk How do you set the measured value if it is a computed non stored field?
_compute_shipping_weight returns the theoretical weight. When you use a scale, you need to be able to enter the value

Reply:
this is not a new field, it's already stored. Here I make it computed and keep stored by setting readonly=false

Reply:
Ah. I didn't know that this implied store=True automatically

Reply:
I didn't know neither, thanks for the info